docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>Consider replacing the 1st paragraph with:</p><p><br></p><p>OOP has been taught in school as the de-facto design pattern to be used throughout the development stack, from web frontend to mobile to backend.</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>